src/DI/Statement.php

```diff
@@ -12,6 +12,8 @@
 
 /**
  * Assignment or calling statement.
+ *
+ * @property $entity
  */
```

Comment:
@dg Like this? Does SmartObject support multiple types? (I didn't find such case it in nette/utils tests.)

```
@property string|array|ServiceDefinition|NULL $entity
```